backend/tests/denver_county_raw.py

```diff
@@ -0,0 +1 @@
rage Household Size\":2.23,\"Median Household Income\":45501,\"Households below 200% FPL\":99021,\"% Households below 200% FPL\":0.39,\"EA Participants\":\"3594\",\"LEAP Participants\":\"15926\",\"Total Households Assisted\":19520,\"% of Households below 200% FPL Assisted\":0.2},\"2011\":{\"Population\":590507,\"Households\":258132,\"Average Household Size\":2.24,\"Median Household Income\":47499,\"Households below 200% FPL\":99932,\"% Households below 200% FPL\":0.39,\"EA Participants\":\"2957\",\"LEAP Participants\":\"12937\",\"Total Households Assisted\":15894,\"% of Households below 200% FPL Assisted\":0.16},\"2012\":{\"Population\":604356,\"Households\":261836,\"Average Household Size\":2.26,\"Median Household Income\":49091,\"Households below 200% FPL\":101707,\"% Households below 200% FPL\":0.39,\"EA Participants\":\"4530\",\"LEAP Participants\":\"12509\",\"Total Households Assisted\":17039,\"% of Households below 200% FPL Assisted\":0.17},\"2013\":{\"Population\":619297,\"Households\":266069,\"Average Household Size\":2.27,\"Median Household Income\":50313,\"Households below 200% FPL\":103765,\"% Households below 200% FPL\":0.39,\"EA Participants\":\"3232\",\"LEAP Participants\":\"11264\",\"Total Households Assisted\":14496,\"% of Households below 200% FPL Assisted\":0.14},\"2014\":{\"Population\":633777,\"Households\":271054,\"Average Household Size\":2.28,\"Median Household Income\":51800,\"Households below 200% FPL\":104129,\"% Households below 200% FPL\":0.38,\"EA Participants\":\"3479\",\"LEAP Participants\":\"10019\",\"Total Households Assisted\":13498,\"% of Households below 200% FPL Assisted\":0.13},\"2015\":{\"Population\":649654,\"Households\":275795,\"Average Household Size\":2.3,\"Median Household Income\":53637,\"Households below 200% FPL\":101261,\"% Households below 200% FPL\":0.37,\"EA Participants\":\"3779\",\"LEAP Participants\":\"8874\",\"Total Households Assisted\":12653,\"% of Households below 200% FPL Assisted\":0.12},\"2016\":{\"Population\":663303,\"Households\":281072,\"Average Household Size\":2.31,\"Median Household Income\":56258,\"Households below 200% FPL\":98956,\"% Households below 200% FPL\":0.35,\"EA Participants\":\"3168\",\"LEAP Participants\":\"8313\",\"Total Households Assisted\":11481,\"% of Households below 200% FPL Assisted\":0.12},\"2017\":{\"Population\":678467,\"Households\":287262,\"Average Household Size\":2.31,\"Median Household Income\":60098,\"Households below 200% FPL\":95839,\"% Households below 200% FPL\":0.33,\"EA Participants\":\"3236\",\"LEAP Participants\":\"7432\",\"Total Households Assisted\":10668,\"% of Households below 200% FPL Assisted\":0.11},\"2018\":{\"Population\":693417,\"Households\":294358,\"Average Household Size\":2.31,\"Median Household Income\":63793,\"Households below 200% FPL\":92530,\"% Households below 200% FPL\":0.31,\"EA Participants\":\"3181\",\"LEAP Participants\":\"7160\",\"Total Households Assisted\":10341,\"% of Households below 200% FPL Assisted\":0.11}}"
```

Comment:
What about combining the three denver data files into one, and then importing from the one file in tests?

Reply:
Updated this.  I think this makes a lot more sense and is way cleaner.